src/Evolution/EventsAndDenseTriggers/DenseTrigger.hpp

```diff
@@ -119,6 +121,12 @@ class DenseTrigger : public PUP::able {
     return previous_trigger_time_;
   }
 
+  void pup(PUP::er& p) noexcept override {
+    p | next_previous_trigger_time_;
+    p | previous_trigger_time_;
+  }
```

Comment:
(Optional) Half of this function is in the wrong commit.  It won't break bisects and splitting is mildly annoying, so I'll leave it up to you whether to fix it.